packages/devtools_app/lib/src/primitives/custom_pointer_scroll_view.dart

```diff
@@ -276,13 +270,13 @@ class CustomPointerScrollable extends StatefulWidget {
   /// If there is no [Scrollable] in the widget tree above the [context], this
   /// method returns false.
   static bool recommendDeferredLoadingForContext(BuildContext context) {
-    final _ScrollableScope widget = context
+    final _ScrollableScope? widget = context
         .getElementForInheritedWidgetOfExactType<_ScrollableScope>()
-        ?.widget as _ScrollableScope;
+        ?.widget as _ScrollableScope?;
     if (widget == null) {
       return false;
     }
-    return widget.position.recommendDeferredLoading(context);
+    return widget.position!.recommendDeferredLoading(context);
```

Comment:
position should not be nullable. Referencing the flutter framework from which a lot of this class was copied https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/scrollable.dart#L366